ceph.spec.in

```diff
@@ -90,6 +91,8 @@
 # disable dwz which compresses the debuginfo
 %global _find_debuginfo_dwz_opts %{nil}
 
+# for jaeger testing
+%define _unpackaged_files_terminate_build 0
```

Comment:
shall we drop this setting?

Reply:
I am afraid we might not have any other way around, since we are building jaeger dependencies with their header files as a dependency too( only needed during build time ) we are not packaging them, which without this option leads to build failure.

Reply:
please remove those header files in `%install` then.